docs/project-setup/filesystem-sync.md

```diff
@@ -60,19 +41,79 @@ services:
+
+The name of the sync volume can be set in a variety of ways. It is determined using the following precedence:
+
+1. Argument provided to the `project sync:start` command
+2. Specified in the `sync` -> `volume-name` property of your [Outrigger Project Configuration](./project-configuration.md)
+3. In your `docker-compose.yml` file, specified as an `external` volume with a name of the pattern `*-sync`
+4. If not specified anywhere else, it will use the name of the current project folder with `-sync` appended
+
+We recommend using approach #2 or #3 to ensure consistency across all environments. If you use approach #2, make sure a volume of the same name is still defined as part of your docker or docker-compose configuration.
+
+### Start it up
 
 From your project root run the command `rig project sync:start` (there is also an alias `rig project sync`) to get going.  
-With the above configuration you should be up and running. The initial sync can take a few seconds depending on the size
-of your project folder. You will see a progress indicator that will let you know when the initial sync is finished and 
+The initial sync can take a few seconds depending on the size
+of your project folder. You will see a progress indicator that will let you know when the initial sync is finished and
 things are ready to use.
 
+!!! important "Workflow Change for Unison-using Projects"
+    Projects that use this file syncing approach will require execution of this command as a new step before any other docker commands can be run.
 
-## Cleaning up
+### Cleaning up
 
 Since the `project sync:start` command starts another container to manage the volume syncing, we have a command `project sync:stop`
-that will clean up that running container for you. It discovers the volume / container name the same way `project sync:start` 
+that will clean up that running container for you. It discovers the volume / container name the same way `project sync:start`
 does.
 
 When you are done for the day, or for that project, from the project root `rig project sync:stop` will clean up any running
 `unison` containers for that project.
+
+## Skipping File Sync on Linux Environments
+
+Linux environments have native filesystem performance without anything as complex as file sync, because they do not need to use a virtualization layer to have access to the [Linux Kernel's container support](https://jvns.ca/blog/2016/10/10/what-even-is-a-container/).
+
+For this reason, any environment running on Linux may want to skip the overhead of the synchronization process.
+
+If you have this use case, create a  docker-compose.override.yml file to carry the external volume declaration, as described in the [official documentation for sharing docker-compose configuration](https://docs.docker.com/compose/extends/#multiple-compose-files). Unlike that documentation, on Linux environments you simply run as `docker-compose -f docker-compose.yml` and the configuration override will not be applied.
```

Comment:
Yes, this is the how-to on how to set this up to support Linux users instead of just assuming always use filesync.